File: src/main/java/me/miki/shindo/management/mods/impl/NametagMod.java
package me.miki.shindo.management.mods.impl;

import lombok.Getter;
import me.miki.shindo.management.language.TranslateText;
import me.miki.shindo.management.mods.Mod;
import me.miki.shindo.management.mods.ModCategory;

public class NametagMod extends Mod {

    @Getter
    private static NametagMod instance;

    public NametagMod() {
        super(TranslateText.NAMETAG, TranslateText.NAMETAG_DESCRIPTION, ModCategory.PLAYER);

        instance = this;
    }

}

File: src/main/kotlin/me/miki/shindo/management/mods/impl/NametagMod.kt
package me.miki.shindo.management.mods.impl

import me.miki.shindo.management.language.TranslateText
import me.miki.shindo.management.mods.Mod
import me.miki.shindo.management.mods.ModCategory
import me.miki.shindo.management.nanovg.font.Shinconic

class NametagMod : Mod(TranslateText.NAMETAG, TranslateText.NAMETAG_DESCRIPTION, ModCategory.PLAYER, Shinconic.MOD_NAMETAG) {
    init {
        instance = this
    }

    companion object {
        @JvmField
        var instance: NametagMod? = null
    }
}
